mv3/addon/lib/request-modifier.ts

```diff
@@ -0,0 +1,122 @@
+import { sameGateway } from "./ipfs-path";
+import debug from "debug";
+const log = debug("ipfs-companion:request");
+log.error = debug("ipfs-companion:request:error");
+
+export default function createRequestModifier(getState, runtime) {
```

Comment:
⭐ MV3 Specific ⭐ This is how we are creating the request from chrome to local node